nextflow_schema.json

```diff
@@ -376,6 +376,11 @@
                     "default": false,
                     "description": "Skip primer trimming with cutadapt. This is not recommended! Use only in case primer sequences were removed before and the data does not contain any primer sequences."
                 },
+                "skip_dada_quality": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Skip quality check with DADA2. This is not only allowed when `--trunclenf` and `--trunclenr` are set."
```

Comment:
"This is not only allowed" feels ambiguous. Perhaps: "Can only be skipped"?

Reply:
You are right!